test/kdump_test.rb

```diff
@@ -729,4 +729,29 @@
       expect(memory).to eq(high: "64", low: "32")
     end
   end
+
+  describe ".AutoYaST" do
+    before do
+      Yast::Mode.SetMode(mode)
+    end
+
+    context "during profile import" do
+      let(:mode) { "autoinstallation" }
+      let(:profile) { {"add_crash_kernel"=>true,
+                        "crash_kernel"=>"256M",
+                        "general"=>{"KDUMP_SAVEDIR"=>"file:///var/dummy"}
+                      }
+                    }
+      # bnc#995750
+      it "imported values will not be overwritten by the proposal" do
```

Comment:
NP: try to use sentences that would read nicely with "it". Like
`it "does not override imported values with the proposal" do`